### PR TITLE
Adding twitter feed for user

### PR DESCRIPTION
**Pull request by vonbarown**
Adding in package for grabbing users twitter feed

### CODE REVIEW

File: pages/twitterFeed/[screenName].js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import { Timeline } from 'react-twitter-widgets';
+export async function getServerSideProps(context) {
+
```
Comment (jenesh) [inline pages/twitterFeed/[screenName].js]: Some weird spacing here but not a biggie.

File: pages/twitterFeed/[screenName].js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  return {
+    props: {
+      screenName: context.params.screenName,
+    },
+  };
+}
+
+const TwitterFeed = ({ screenName }) => {
+  console.log(screenName);
```
Comment (jenesh) [inline pages/twitterFeed/[screenName].js]: We can take out consoles unless you want them for further changes.

**Review by jenesh — APPROVED** [review]
Nice and easy twitter feed. LGTM 🚀

File: pages/twitterFeed/[screenName].js (hunk 2 above)
Comment (vonbarown) [reply pages/twitterFeed/[screenName].js]: gonna take it out forgot about it

**Pull request merged**